generators/ALIENStarGenerator.js

```diff
@@ -10,9 +9,11 @@ const Util = require('../utils/Util');
 class ALIENStarGenerator extends YZGenerator2 {
 	/**
 	 * Defines a star.
+	 * @param {string} language Language code of the language to be used
```

Comment:
```suggestion
	 * @param {string} language Language code to be used
```

Reply:
I think i used a different sentence depending on the day i did it 🙈